src/content/structured/components/side-nav/code.mdx

```diff
@@ -585,128 +565,68 @@ export const snippetsExpanded = [
-                  fill="currentColor"
-                />
-              </svg>
-              Home
-            </NavLink>
-          </IcNavigationItem>
-          <IcNavigationGroup label="Navigation Grouped" slot="primary-navigation" expandable="true">
-            <IcNavigationItem>
-              <NavLink to="/child-item" slot="navigation-item">
-                <svg
-                  slot="icon"
-                  width="24"
-                  height="24"
-                  viewBox="0 0 24 24"
-                  fill="none"
-                  xmlns="http://www.w3.org/2000/svg"
-                >
-                  <path
-                    d="M12 6.19L17 10.69V18.5H15V12.5H9V18.5H7V10.69L12 6.19ZM12 3.5L2 12.5H5V20.5H11V14.5H13V20.5H19V12.5H22L12 3.5Z"
-                    fill="currentColor"
-                  />
-                </svg>
-                Home
-              </NavLink>
-            </IcNavigationItem>
-            <IcNavigationItem>
-              <NavLink to="/child-item-2" slot="navigation-item">
-                <svg
-                  slot="icon"
-                  width="24"
-                  height="24"
-                  viewBox="0 0 24 24"
-                  fill="none"
-                  xmlns="http://www.w3.org/2000/svg"
-                >
-                  <path
-                    d="M12 6.19L17 10.69V18.5H15V12.5H9V18.5H7V10.69L12 6.19ZM12 3.5L2 12.5H5V20.5H11V14.5H13V20.5H19V12.5H22L12 3.5Z"
-                    fill="currentColor"
-                  />
-                </svg>
-                Daily Tippers
-              </NavLink>
-            </IcNavigationItem>
-          </IcNavigationGroup>
-          <IcNavigationItem
-            slot="secondary-navigation"
-          >
-            <NavLink to="/daily-tippers" slot="navigation-item">
-              <svg
-                slot="icon"
-                width="24"
-                height="24"
-                viewBox="0 0 24 24"
-                fill="none"
-                xmlns="http://www.w3.org/2000/svg"
-              >
-                <path
-                  d="M12 6.19L17 10.69V18.5H15V12.5H9V18.5H7V10.69L12 6.19ZM12 3.5L2 12.5H5V20.5H11V14.5H13V20.5H19V12.5H22L12 3.5Z"
-                  fill="currentColor"
-                />
-              </svg>
-                Daily Tippers
-            </NavLink>
-          </IcNavigationItem>
-        </IcSideNavigation>
-        <main>
-          <IcSectionContainer>
-            <Routes>
-              <Route path="/" element={() => <IcTypography>This is the home page</IcTypography>} />
-              <Route path="/daily-tippers" element={() => <IcTypography>This page is about daily tippers</IcTypography>} />
-            </Routes>
-          </IcSectionContainer>
-        </main>
-      </div>
-    </MemoryRouter>
+  <div style={{ display: "flex", flexDirection: "row"}}>
+    <IcSideNavigation
+      appTitle="ACME"
+      version="v0.0.0"
+      status="BETA"
```

Comment:
can you just check this one on smaller device sizes? - i think it might need disableAutoParentStyling applying - it looks indented otherwise